src/ompl/geometric/planners/fmt/src/FMT.cpp

```diff
@@ -478,10 +581,12 @@ bool ompl::geometric::FMT::expandTreeFromNode(Motion **z)
                 x->setParent(yMin);
                 x->setCost(cMin);
                 x->setHeuristicCost(opt_->motionCostHeuristic(x->getState(), goalState_));
+                yMin->getChildren().push_back(x);
+
                 // Add x to Open
-                Open_new.push_back(x);
-                // Remove x from Unvisited
-                x->setSetType(Motion::SET_CLOSED);
+                Open_.insert(x);
```

Comment:
I remember we discussed this a long time ago as a negligible difference from the paper version of FMT (see http://arxiv.org/pdf/1306.3532.pdf, footnote 2, page 8). I can't remember - was the timing difference negligible as well?